ashlar/imports/shapefile.py

```diff
@@ -21,7 +21,7 @@ def get_shapefiles_in_dir(path):
     for dirpath, subdirs, files in os.walk(path):
         for name in files:
             all_files.append(os.path.join(dirpath, name))
-    shapefiles = filter(lambda f: f.endswith('.shp'), all_files)
+    shapefiles = [f for f in all_files if f.endswith('.shp')]
```

Comment:
`filter` returns a list in Python 2, but returns a generator in Python 3. Using a list comprehension makes the expected data type more explicit.